server/belga/io/feed_parsers/base_belga_newsml_1_2.py

```diff
@@ -497,35 +542,34 @@ def parser_descriptivemetadata(self, item, descript_el):
         subjects += descript_el.findall('SubjectCode/Subject')
         item.setdefault('subject', []).extend(self.format_subjects(subjects))
 
-        # parser OfInterestTo element
-        elements = descript_el.findall('OfInterestTo')
-        if elements:
-            item['OfInterestTo'] = []
-            for element in elements:
-                item['OfInterestTo'].append(element.get('FormalName', ''))
+        # parser OfInterestTo is CV
+        for element in descript_el.findall('OfInterestTo'):
+            if element.get('FormalName'):
+                # genre CV
+                item.setdefault('subject', []).append({
+                    "name": element.get('FormalName'),
+                    "qcode": element.get('FormalName'),
+                    "scheme": "of_interest_to"
+                })
 
         element = descript_el.find('DateLineDate')
         if element is not None:
-            if 'dayline' in item:
-                item['dateline']['date'] = self.datetime(element.text)
-            else:
-                item['dateline'] = {}
-                item['dateline']['date'] = self.datetime(element.text)
+            item.setdefault('dateline', {}).update({"date": self.datetime(element.text)})
 
         location_el = descript_el.find('Location')
         if location_el is not None:
-            item['location'] = {}
+            item['extra'] = {}
 
             how_present_el = location_el.get('HowPresent', '')
             if how_present_el is not None:
-                item['location']['how_present_el'] = how_present_el
+                item['extra']['how_present'] = how_present_el
 
             elements = location_el.findall('Property')
             for element in elements:
                 if element.attrib.get('FormalName', '') == 'Country':
-                    item['location']['country'] = element.attrib['Value']
+                    item['extra']['country'] = element.attrib['Value']
```

Comment:
I would use `.get` here to prevent raising error, `Value` attribute might be absent. For example: 
```
<Location>
            <Property FormalName="City" Value="BRUXELLES"/>
            <Property FormalName="Country"/>
            <Property FormalName="CountryArea"/>
            <Property FormalName="SubLocation"/>
            <Property FormalName="WorldRegion"/>
          </Location>
```